src/power_grid_model_io/converters/base_converter.py

```diff
@@ -92,7 +92,7 @@ def load_asym_output_data(self, data: Optional[T] = None) -> Dataset:
         data = self._load_data(data)
         return self._parse_data(data=data, data_type="asym_output", extra_info=None)
 
-    def convert(self, data: Dataset, extra_info: Optional[ExtraInfoLookup] = None) -> T:
+    def convert(self, data: Dataset, extra_info: Optional[ExtraInfoLookup] = None, **kwargs: Optional[str]) -> T:
```

Comment:
I think **kwargs can be for `Any` type. `Any` includes `None`, so it it per definition `Optional`.